src/components/main/main-navbar.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import arrow from '../../images/main navbar/arrow.png';
+
+const MainNavbar = () => {
+  return (
+    <div className="main-navbar">
+      <div clas="main-navbar-open-aside">
+        {/* <span style="fontSize:30px;cursor:pointer" onClick="openNav()">&#9776;</span>         */}
+      </div>
+      <div className="search-box">
+        <input className="search-txt" type="text" placeholder="Search..." />
+        <a className="search-btn" href="searchButton">
+          <button className="search-btn" type="button">
+            <img src={arrow} alt="arrow" />
+          </button>
+        </a>
+      </div>
+      <div className="navbar-logo">
+        <h4>glazzed</h4>
+      </div>
+    </div>
+  );
+};
+
+export default MainNavbar;
```

Comment:
do we really need default exports in all these cases?

Reply:
eslint say that I must to use default props